### PR TITLE
add fablab barcelona barduino

### PR DESCRIPTION
**Pull request by FoamyGuy**

### CODE REVIEW

File: _board/barduino.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+date_added: 2024-09-06
+family: esp32s3
+features:
+  - Breadboard-Friendly
+  - USB-C
+  - Bluetooth/BTLE
+  - Wi-Fi
+---
+
+Barduino is a development board created by Fablab Barcelona as a tool for it's educational programs.
```
Comment (dhalbert) [inline _board/barduino.md:19]:
```suggestion
Barduino is a development board created by Fablab Barcelona as a tool for its educational programs.
```

**Review by dhalbert — CHANGES_REQUESTED** [review]
Just a typo.

File: _board/barduino.md (hunk 1 above)
Comment (FoamyGuy) [reply _board/barduino.md]: Nice catch, thank you. committed that fix.

**Review by tannewt — APPROVED** [review]
Thank you!

**Review by dhalbert — APPROVED** [review]
:+1:

**Pull request merged**